cardano-api/src/Cardano/Api/Query.hs

```diff
@@ -496,6 +535,12 @@ fromConsensusQueryResult QuerySystemStart q' r' =
         -> r'
       _ -> fromConsensusQueryResultMismatch
 
+fromConsensusQueryResult QueryHeaderStateTip q' r' =
+    case q' of
```

Comment:
Maybe it'd be easier to read if the part after the `of` would be aligned in 2 lines.

Reply:
I hesitate to break with the existing formatting.  I'll let my team decide if reformatting is warranted here.